sunpy/image/coalignment.py

```diff
@@ -504,7 +505,6 @@ def calculate_match_template_shift(mc, template=None, layer_index=0,
     return {"x": xshift_arcseconds, "y": yshift_arcseconds}
 
 
-# Coalignment by matching a template
 def mapcube_coalign_by_match_template(mc, template=None, layer_index=0,
```

Comment:
this needs a `@deprecated` decorator on it and then it could call the new function.